pkg/cache/groupcache.go

```diff
@@ -135,6 +132,12 @@ func NewGroupcacheWithConfig(logger log.Logger, reg prometheus.Registerer, conf
 
 			switch parsedData.Verb {
 			case cachekey.AttributesVerb:
+				_, attrCfg := cfg.FindAttributesConfig(parsedData.Name)
+				if attrCfg == nil {
+					// TODO: Debug this. Why? Attributes get called for Chunks.
```

Comment:
:thinking: I wonder why. How to reproduce this?

Reply:
TO reproduce this, you can run the test `TestStoreGatewayGroupCache`.

Reply:
It seems like we always get the attributes in GetRange: https://github.com/thanos-io/thanos/blob/main/pkg/store/cache/caching_bucket.go#L329-L336. Presumably, this is because internally the caching bucket divides each given range into multiple smaller subranges so we need to know the length of the object to do this. I was able to make the tests pass with:

```go
groupcacheCfg.CacheAttributes("chunks", c, isTSDBChunkFile, config.ChunkObjectAttrsTTL)
```

Could you add it?

Reply:
Yes, thanks!